model/project_vars.go

```diff
@@ -216,11 +216,23 @@ func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
 	return restrictedVars
 }
 
+func (projectVars *ProjectVars) GetAdminOnlyVars() map[string]string {
+	adminOnlyVars := map[string]string{}
+
+	for k, v := range projectVars.Vars {
+		if projectVars.AdminOnlyVars[k] {
+			adminOnlyVars[k] = v
+		}
+	}
+	return adminOnlyVars
+}
+
+// Returns non-restricted and non-admin only vars for now.
```

Comment:
nit: function comments should start with the name of the function

Reply:
Can you link the ticket where this will be changing (probably where you'll be removing restricted) rather than for now?

Reply:
done